data/fields/sidewalk.json

```diff
@@ -0,0 +1,35 @@
+{
+    "keys": [
+        "sidewalk"
+    ],
+    "reference": {
+        "key": "sidewalk"
+    },
+    "type": "sidewalk",
```

Comment:
This isn’t a valid type – unless the intention was to introduce a new type to schema-builder? Perhaps the `cycleway` type used by cycleway.json was replaced by `sidewalk` without realizing the purpose of `type`. But cycleway.json implements `cycleway:left/right=*`, only setting `cycleway=*` if the left and right values agree. It would be possible to piggyback on the `cycleway` type, but the only `options` would be `no`, `yes`, and `separate`, and `sidewalk:left` and `sidewalk:right` would need to be defined as `types`.

Reply:
>This isn’t a valid type – unless the intention was to introduce a new type to schema-builder?

Yeah, my intention was to create a new `type` that's similar to the `cycleway` type [as shown in this pdf](https://github.com/openstreetmap/id-tagging-schema/files/8692005/Sidewalk.field.concept.pdf), but I'm wasn't sure how (or if) this could be implemented.